lms/djangoapps/courseware/models.py

```diff
@@ -145,10 +153,11 @@ def __unicode__(self):
         return unicode(repr(self))
 
 
-class StudentModuleHistory(models.Model):
+class StudentModuleHistory(CallStackMixin, models.Model):
```

Comment:
update: Mixin has no effect if we have it after `Model.models`. Reverted back to original sequence.